src/braket/aws/aws_session.py

```diff
@@ -211,8 +211,7 @@ def create_quantum_task(self, **boto3_kwargs) -> str:
         Create a quantum task.
 
         Args:
-            ``**boto3_kwargs``: Keyword arguments for the Amazon Braket `CreateQuantumTask`
-              operation.
+            `**boto3_kwargs`: Keyword arguments for the Amazon Braket `CreateQuantumTask` operation.
```

Comment:
You can leave the double `` here and below.

Reply:
Fixed, thanks!